man/systemd.network.xml

```diff
@@ -1372,6 +1372,22 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>PDOtherInformationQuirk=</varname></term>
```

Comment:
Can't say I like the name.

Reply:
Me neither. There is still ample time to name this setting, the route setting and unsetting issues noticed by @vanaf have been fixed but I have to call it a day and update the code according to the above review comments tomorrow...

Reply:
I changed the name to 'ForceDHCPv6PDOtherInformation', still too long but perhaps a bit better...

Reply:
Not sure why this killswitch is needed.
PD will be sent only if some interfaces will have PrefixDelegation set to yes or dhcp.
Are we think someone who set this PrefixDelegation option dont want PD to be sent on other flag? I believe most of users dont know what flags are sent by ISP, they just want PD to work

Reply:
Or we want to have option to disable PD requests sent on some interaces. (If we have multiple interfaces with dhcpv6 on them) 
Why in this case we want to enable/disable it only for other flag? Maybe we should just killswitch what requests sent, no matter how it was originated, by other or by managed flags.

Reply:
Usually we do send only Information-Request messages when the 'O' bit is set. So this forces stateful messages to be sent even without the needed 'M' bit. I hope it works as intended this way, the mentioned RFC doesn't really make things clearer here.